xapian-bindings/python3/docs/examples/xapian-train.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+#
+# Simple example script demonstrating how to train and save the model.
+#
+# Copyright (C) 2019 Vaibhav Kansagara
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301
+# USA
+
+import sys
+import xapian
+import xapianletor
+
+# We require at least two command line arguments.
+if len(sys.argv) < 3:
+    print("Usage: PATH_TO_DATABASE PATH_TO_TRAINING_FILE",
+          sep='\n', file=sys.stderr)
+    sys.exit(1)
+
+try:
+    # Open the database for searching.
```

Comment:
Again, this isn't what the following lines do.

Reply:
fixed in a fixup commit.